postprocessing/postprocess_auto.py

```diff
@@ -200,7 +200,7 @@ def generate_pdf(
         # In[5]:
 
         # gempyor.config.set_file(run_info.config_filepath)
-        # gt = pd.read_csv(gempyor.config["inference"]["data_path"].get())
```

Comment:
if this is now the correct argument name for inference, need to revise the previous tweaks that totally delete data path to instead refer to gt_data_path.